src/app_handle.rs

```diff
@@ -16,33 +16,30 @@ use crate::{
     },
     event::{Event, EventListener},
     ext_event::EXT_EVENT_HANDLER,
-    id::{Id, IDPATHS},
+    id::{Id, ID_PATHS},
     responsive::ScreenSize,
     style::{CursorStyle, Style},
     view::{ChangeFlags, View},
 };
 
 thread_local! {
+    /// Stores a queue of update messages for each view. This is a list of build in messages, including a built-in State message
+    /// that you can use to send a state update to a view.
     pub(crate) static UPDATE_MESSAGES: std::cell::RefCell<HashMap<Id, Vec<UpdateMessage>>> = Default::default();
     pub(crate) static ANIM_UPDATE_MESSAGES: std::cell::RefCell<Vec<AnimUpdateMsg>> = Default::default();
+    ///
     pub(crate) static DEFERRED_UPDATE_MESSAGES: std::cell::RefCell<DeferredUpdateMessages> = Default::default();
 }
 
 pub type FileDialogs = HashMap<FileDialogToken, Box<dyn Fn(Option<FileInfo>)>>;
 type DeferredUpdateMessages = HashMap<Id, Vec<(Id, Box<dyn Any>)>>;
 
-pub struct AppHandle<V: View> {
-    scope: Scope,
-    view: V,
-    handle: glazier::WindowHandle,
-    app_state: AppState,
-    paint_state: PaintState,
-
-    file_dialogs: FileDialogs,
-}
-
+// Primarily used to mint and assign a unique ID to each view.
+// It also contains a constant scope which is used to create signals.
+// TODO: would it be better to call this `TreeContext`? Since it is really only about minting Ids given the current position in the tree.
```

Comment:
I'd like to call this something like `TreeContext` since it is really only about minting Ids given the current position in the tree.